build-logic/src/main/kotlin/adventure.common-conventions.gradle.kts

```diff
@@ -85,7 +85,7 @@ dependencies {
   testImplementation("com.google.guava:guava-testlib:30.1.1-jre")
   testImplementation("com.google.truth:truth:1.1.3")
   testImplementation("com.google.truth.extensions:truth-java8-extension:1.1.3")
-  testImplementation(platform("org.junit:junit-bom:5.7.2"))
+  testImplementation(platform("org.junit:junit-bom:5.8.2"))
```

Comment:
this is incorrect -- will need to restrict to first-level contents of build-logic and buildSrc